management/version.go

```diff
@@ -0,0 +1,5 @@
+package management
+
+var (
+	sdkVersion = "8.0.0-beta"
```

Comment:
Ugh, ideally I would have wanted to skip reporting these broken version numbers. However, this is a chicken & egg problem. When I refactor out management into its own repository, I'll have to remember to change the UserAgent string to "Azure-SDK-For-Go-ASM/<updatedVersionNumber>" so that we can differentiate between this version and the refactored version.